README.md

```diff
@@ -59,73 +59,73 @@ parameter to the `puppet_enterprise::profile::server` class with a string value
 
    To use SSL verification so that the Satellite server can verify the Puppet server, you must generate a SSL cert and key pair on the Satellite server, and then copy these files to your Puppet server.
 
-> Note: In the following steps, replace `puppet.example.com` with the FQDN of your Puppet server.
+  > Note: In the following steps, replace `puppet.example.com` with the FQDN of your Puppet server.
 
-a. On the Satellite server, run the following command:
+  1. On the Satellite server, run the following command:
 
-```
-capsule-certs-generate --capsule-fqdn "puppet.example.com" \
---certs-tar "~/puppet.example.com-certs.tar"
-```
-> Note: Use `--foreman-proxy-fqdn` instead of `--capsule-fqdn` for Satellite 6.3
+            capsule-certs-generate --foreman-proxy-fqdn "puppet.example.com" \
+            --certs-tar "~/puppet.example.com-certs.tar"
 
-b. Untar the newly created file:
+       > Note: Use `--capsule-fqdn` instead of `--foreman-proxy-fqdn` for Satellite versions older than 6.3
 
-```
-tar -xvf ~/puppet.example.com-certs.tar
-```
+  2. Untar the newly created file:
 
-This creates a new folder: `~/ssl-build`. This may contain either raw `.crt` and `.key` file, or a number of RPM files.
+            tar -xvf ~/puppet.example.com-certs.tar
 
-c. If the ssl-build folder contains RPM files for the host, find and extract the contents of the puppet-client rpm file:
+     This creates a new folder: `~/ssl-build`. This may contain either raw `.crt` and `.key` file, or a number of RPM files.
 
-```
-cd ~/ssl-build/puppet.example.com
-rpm2cpio puppet.example.com-puppet-client-1.0-1.noarch.rpm | cpio -idmv
-```
+  3. If the ssl-build folder contains RPM files for the host, find and extract the contents of the puppet-client rpm file:
 
-This creates a folder structure in the current directory beginning with `./etc/pki/katello-certs-tools/`
+            cd ~/ssl-build/puppet.example.com
+            rpm2cpio puppet.example.com-puppet-client-1.0-1.noarch.rpm | cpio -idmv
```

Comment:
The produced RPMs are deprecated and will be dropped in a future release.

Reply:
Until line 121, README changes in this commit are just for markdown formatting. I do not mean to suggest any content changes in this PR outside of the additions starting on line 121.

Reply:
Consider it a talking point for our chat.